.github/dependabot.yaml

```diff
@@ -8,5 +8,5 @@ updates:
     interval: "monthly"
   groups:
     github-actions:
-    patterns:
-    - "*"
+      patterns:
+      - "*"
```

Comment:
nit: Although both are accepted by yaml, probably [adding space for array items is mor ereadable](https://stackoverflow.com/questions/17014460/yaml-indentation-for-array-in-hash), also as in the example [here](https://docs.github.com/en/code-security/dependabot/dependabot-security-updates/configuring-dependabot-security-updates#overriding-the-default-behavior-with-a-configuration-file).